src/server/journal/serializer.h

```diff
@@ -45,6 +45,9 @@ struct JournalReader {
   // Initialize start database index.
   JournalReader(DbIndex dbid);
 
+  // Overwrite current db index.
+  void SetDb(DbIndex dbid);
```

Comment:
I guess we are not using now the logic of the db index in the constructor, if so it can be dropped.